core/tx_pool.go

```diff
@@ -1187,14 +1166,10 @@ func (pool *TxPool) reset(oldHead, newHead *types.Header) {
 	pool.pendingNonces = newTxNoncer(statedb)
 	pool.currentMaxGas = newHead.GasLimit
 
-	// OVM Change. Do not reinject reorganized transactions
-	// into the mempool.
-	if vm.UsingOVM {
-		// Inject any transactions discarded due to reorgs
-		log.Debug("Reinjecting stale transactions", "count", len(reinject))
-		senderCacher.recover(pool.signer, reinject)
-		pool.addTxsLocked(reinject, false)
-	}
+	// Inject any transactions discarded due to reorgs
+	log.Debug("Reinjecting stale transactions", "count", len(reinject))
```

Comment:
This runs as part of our hot path, list logline shows up. Its all useless code that executes - at minimum we should remove this logline, maybe we can return from this function earlier to save a bit of useless compute